extensions/migadu/package.json

```diff
@@ -56,12 +59,19 @@
       "subtitle": "Migadu",
       "description": "View, Create, Update or Delete Rewrites in your Migadu Account",
       "mode": "view"
+    },
+    {
+      "name": "create-copy-anonymous-identity",
+      "title": "Create Anonymous Identity (Custom Hide My Email)",
```

Comment:
I chose this lengthy name so that users can search for "Hide My Email" and find this command. If it's too long, maybe we can find another way to enable that?

Reply:
We can use `keywords` for a command (https://developers.raycast.com/information/manifest#command-properties) but the title seems OK to me.